src/utils/syncSpace.js

```diff
@@ -33,6 +33,20 @@ export const isSpaceUpToDate = (localSpace, remoteSpace) => {
   return _.isEqual(filteredLocalSpace, filteredRemoteSpace);
 };
 
+// return whether the localSpace is different from the remoteSpace
+// handle immutable spaces
+// @param localSpace : saved space in local database, immutable Map
```

Comment:
Getting fancy with the documentation! For these function-level descriptions, I don't have a best practice yet. But often I see them wrapped in `/** */` multiline comments. I guess we can follow JSDoc, if you want.

https://gomakethings.com/whats-the-best-way-to-document-javascript/